test/libsolidity/syntaxTests/string/string_concat_with_parameters_as_bytes.sol

```diff
@@ -0,0 +1,11 @@
+contract C {
```

Comment:
Would be good to have some more test cases, e.g. for fixed bytes types and number literals as arguments.